lib/caesar.rb

```diff
@@ -1,6 +1,34 @@
 class Caesar
-  attr_reader :key
+  attr_reader :phrase
 
-  def initialize
+  def initialize(phrase, shift = 0)
```

Comment:
Read the spec more carefully. You will see that our Caesar object is not initialized with any parameters. The phrase and shift are passed to the `eng_to_cipher` method